wiki/Website-Changelog.md

```diff
@@ -0,0 +1,172 @@
+```
+
+#### Multiple changes per element
+```
+**`{component-name}`** - Changed {...additional details}, including:
+- {change 1}
+- {change 2}
+- etc
+
+_{additional details}..._
+```
+
+#### Breaking changes
+```
+**`{component-name}`** - ⛔️ **Breaking change** - Changed {...additional details}.
+
+_{additional details}..._
+```
+
+<details>
+  <summary><h4>Example outputs for npm packages</h4></summary>
+  <i>These examples are just outputs of the details provided as a part of the changeset. They do not include the PR links or contributors, which are added automatically.</i>
+  <br/><br/>
+  
+  **`TooltipButton`** - Added `text-align: inherit` to the “button” element.
+  
+  **`Dropdown`** - Fixed a few accessibility failures, including:
+  - Added `aria-hidden` to the separator,
+  - Added `role=“none”` to the checkbox `<li>`.
+    
+  **`Table`** - Changed `height` to `min-height` for the table head cells and updated the cells' internal padding to align with the design specs in Figma.
+  
+  **`Sidenav Link`** - Fixed an issue resulting in an empty node.
+  - This will lead to a minimal visual impact on some edge cases of `Alert` and `Toast` (multiple description items) and `Sidenav` (text and generic content).
+  
+  **`Standalone Link`** - Increased the space between the actions to 16px and reduced the size of the icon in the small variant to 12px.
+  **`Modal`** and **`Flyout`** - Fixed the scroll management resulting in stray `style` attribute on the `<body>` element.
+  Updated the CSS of multiple components to use flex `gap`.
+  Upgraded Ember.js to the latest stable release 4.12, including upgrades to:
+  - `ember-auto-import` from `2.6.0` to `2.6.3`
+  - `ember-cli-htmlbars` from `6.1.0` to `6.2.0`
+  
+  Upgraded the following dependencies:
+  - `ember-focus-trap` from `1.0.1` to `1.0.2`
+  - `ember-keyboard` from `8.1.0` to `8.2.0`
+  - `ember-truth-helpers` from `3.0.0` to `3.1.1`
+  - `sass` from `1.58.3` to `1.62.1`
+  
+  Shifted our support version of Node.js from `12.* || 14.* || >= 16` to `14.* || 16.* || >= 18`
+</details>
+
+
+### Templates for Figma changelog entries
+
+Entries made for the Figma Components and Foundations UI Kits require a few additional steps. Copy and paste this template and adjust as necessary after publishing changes in Figma. _Replace all elements in brackets._
+
+#### Simple entry
+```
+### {month day, year}
+
+**`{component-name}`** - Fixed {...additional details}.
+```
+
+#### Multiple changes per element
+```
+### {month day, year}
+
+**`{component-name}`** - Changed {...additional details}, including:
+- {change 1}
+- {change 2}
+- etc
+
+_{additional details}..._
+```
+
+#### Breaking changes
+```
+### {month day, year}
+
+**`{component-name}`** - ⛔️ **Breaking change** - Changed {...additional details}.
+
+_{additional details}..._
+```
+
+<details>
+  <summary><h4>Example outputs for Figma changes</h4></summary>
+
+  <h3>September 15, 2023</h3>
```

Comment:
```suggestion
  <h3>Sep 15, 2023</h3>
```

Reply:
Could we consider abbreviating the month names, similar to the format used in the Changelog in Figma? This change could help create a consistent visual pattern within the sidecar.

Reply:
I agree that there should be a consistent pattern, but I disagree with it being the abbreviated version. Not all screen readers will interpret the abbreviated version as the month, instead reading it out at "Sep". Here are a few examples of how dates are read out to some screen readers: https://pauljadam.com/demos/dates.html